### DISCUSSION TITLE
Intel quadcore Q6600 no avx

### DISCUSSION THREAD

**Discussion by ionultd** [blakeblackshear/frigate#1031] [Ask A Question]
Tried installing on this cpu but it give an install error dialog and quits. Any suggestions?

**Comment by pdecat** [comment]
Hi, have a look at https://github.com/blakeblackshear/frigate/issues/49

I adapted an _old CPU_ variant of the addon: https://github.com/pdecat/frigate-hass-addons

Which I'm trying to upstream with https://github.com/blakeblackshear/frigate-hass-addons/pull/12

**Reply by ionultd** [reply]
Ok wow cool. SO how do I get this addon version? BTW I do not yet have a Coral Board.
